R/cocoProblemInitObserver.R

```diff
@@ -0,0 +1,36 @@
+#' @title Init coco observer.
+#'
+#' @description
+#' See title
```

Comment:
i would add 2 lines here what an observer actually does and is for. 
also a space before next at_param